ivy/functional/frontends/numpy/indexing_routines/inserting_data_into_arrays.py

```diff
@@ -28,3 +29,12 @@ def fill_diagonal(a, val, wrap=False):
     temp = ivy.flatten(a)
     temp[:end:step] = val
     a = ivy.reshape(temp, shape)
+
+
+@to_ivy_arrays_and_back
+def putmask(a, mask, values, /):
+    if values.size == 0:
+        return
+    if values.size != a.size:
+        values = np_frontend.resize(values, a.shape)
+    a[mask] = values[mask]
```

Comment:
I think you forgot the `return` for this function

Reply:
According to the documentation, the function doesn't return anything rather changes the array in place. [Here](https://numpy.org/doc/stable/reference/generated/numpy.putmask.html) is the relevant link